harmony-fe/src/pages/register/register.tsx

```diff
@@ -48,11 +58,16 @@ const RegisterPage = () => {
       console.error('Error:', error)
       setHasError(true)
     } else if (!loading) {
+      setHideSignUp(true)
       document.cookie = response['Set-Cookie']
       window.location.href = '../home'
     }
   }, [response, error])
 
+  useEffect(() => {
```

Comment:
I added this to avoid changing loading initial value in httprequest object in case someone is relying on that logic